packages/block-library/src/navigation/index.php

```diff
@@ -15,7 +15,7 @@
 function build_css_colors( $attributes ) {
 	// CSS classes.
 	$colors = array(
-		'css_classes'   => [],
+		'css_classes'   => array(),
```

Comment:
just curious about this change.

Reply:
It's automatically applied by WP Coding Standards linter.